test/aria/widgets/form/autocomplete/popupWidth/AdaptToContentWidthTplScript.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2013 Amadeus s.a.s.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Aria.tplScriptDefinition({
+    $classpath : "test.aria.widgets.form.autocomplete.popupWidth.AdaptToContentWidthTplScript",
+    $dependencies : ["test.aria.widgets.form.autocomplete.popupWidth.CustomResourceHandler"],
+    $destructor : function () {
+        this.handler.$dispose();
+    },
+    $prototype : {
+
+        getHandler : function () {
+            if (!this.handler) {
+                this.handler = new test.aria.widgets.form.autocomplete.popupWidth.CustomResourceHandler(this.$getId('messages'));
```

Comment:
Is the `this.$getId('messages')` parameter really useful? I don't understand where it is used.